swagger.yaml

```diff
@@ -274,9 +274,12 @@ definitions:
 
   ApplicationPatch:
     type: 'object'
+    description: 'One of status OR validations'
     properties:
```

Comment:
@neilkidd Did you decide against the `oneOf` thing?

Reply:
It would be a slightly more significant change and is API 3. So for now I went for this